src/settings/LoanHistory/LoanHistoryForm.js

```diff
@@ -67,9 +79,17 @@ class LoanHistoryForm extends Component {
     );
   }
 
+  // Due to the issue https://github.com/erikras/redux-form/issues/4101, has to use
+  // multiple dispatches of actions instead of single dispatch of 'clearFields' action.
```

Comment:
```
// Due to the issue https://github.com/erikras/redux-form/issues/4101,
// multiple dispatches of actions should be used instead of single dispatch of 'clearFields' action.
```